src/react/components/common/tagColors.json

```diff
@@ -1,6 +1,4 @@
 [
-    "#FFFFFF", 
```

Comment:
I think we actually might be able to get rid of this file entirely now. I had forgotten that it was still there

Reply:
If not, we should update to use the object from `tagColors.ts`

Reply:
I'll try getting rid of it and see if I run into any issues

Reply:
@JacopoMangiavacchi is using the colors somewhere in VoTT. I believe within the TF records

Reply:
Yes I'm using this when importing TFRecords.  In that scenario we are importing potentially new tag definitions that have to be added to IProject.ITags list.  Basically I need to associate a new color from that list to the new ITag object.